docs/ensnode.io/src/styles/starlight.css

```
@@ -916,15 +906,27 @@ starlight-tabs ul[role="tablist"] {
+.static-example starlight-tabs .starlight-aside :not(pre)>code {
   margin-block: -0.125rem;
   padding: 0.125rem 0.375rem;
   font-size: var(--sl-text-code-sm);
 }
+
+.static-example p a {
+  color: var(--sl-color-text-accent);
+  text-decoration: underline;
+  text-underline-offset: 4px;
+
+  transition: text-underline-offset 0.2s ease-in-out;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)